### ISSUE TITLE
shall we disable realign_complexes()

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/deeprank#145]
Since we have a function to align PDB files, shall we disable `realign_complexes()` from hdf5? 

realign_complexes() has the following issues:
1. It needs pssm files as input even though the not-aligned hdf5 file contains features_raw/PSSM_ALA_raw and so on.
2. It deletes the hdf5 file when an error occurs. 

Sorry that we should have asked for this function in the first place.

**Comment by NicoRenaud** [comment]
yes it might be best ... i should have kept it as a quick fix and not part of the module.
